core/embed/projects/prodtest/README.md

```
@@ -1242,20 +1242,17 @@ OK 00000001000000000000000000000000000000000000000000000000000000000000000001000
 ### tropic-update-fw
 
 Updates Tropic firmware to the embedded version.
+Only chips with the ACAB silicon revision are supported.
 
 Example:
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)